gormsupport/cleaner/db_clean.go

```diff
@@ -64,10 +67,10 @@ func DeleteCreatedEntities(db *gorm.DB) func() {
 			entry := entires[i]
 			log.Info(nil, map[string]interface{}{
 				"table":     entry.table,
-				"key":       entry.key,
+				"keys":      entry.keys,
 				"hook_name": hookName,
-			}, "Deleting entities from '%s' table with key %v", entry.table, entry.key)
-			tx.Table(entry.table).Where(entry.keyname+" = ?", entry.key).Delete("")
+			}, "Deleting entities from '%s' table with keys %v", entry.table, entry.keys)
+			tx.Debug().Table(entry.table).Where(entry.keys).Delete("")
```

Comment:
Do you want to leave the Debug() here?

Reply:
I would leave it. The logs are printed only if there is a test failure. But I'm not sure.. maybe there are too many logs in case of failure...

Reply:
No problem, I just wanted to point out that it was here. :)